app/main.py

```diff
@@ -2,0 +1,15 @@
+
+def copy_file(command: str) -> None:
+    command_parsed = command.split(" ")
+    file_in_name = command_parsed[1]
+    file_out_name = command_parsed[2]
+
+    if command_parsed[0] == "cp":
+        if file_in_name == file_out_name:
+            return None
```

Comment:
This line returns None if the source file name is the same as the destination file name. However, it would be better to inform the user about this situation. You could raise an exception or print a message to the console.